openapi/openapi.yaml

```diff
@@ -8509,6 +8500,16 @@ components:
         - error
         - status
         - message
+    EditorType:
+      type: string
+      description: The type of editor.
+      enum:
+        - figma
+        - figjam
+        - slides
+        - buzz
+        - sites
+        - make
```

Comment:
Not sure if I was too aggressive here since I don't know the full Figma product portfolio. If not this then it should just be `[figma, figjam]` for the values.